scss/elements/dialogs.scss

```diff
@@ -0,0 +1,33 @@
+dialog {
+  position: relative;
+  padding: 1.5rem 2rem;
+
+  > :last-child {
+    margin-bottom: 0;
+  }
+
+  &::before,
+  &::after {
+    position: absolute;
+    z-index: -1;
+    content: "";
+  }
+
+  &::before {
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+
+  &::after {
+    top: 2px;
+    right: 2px;
+    bottom: 2px;
+    left: 2px;
+    border-color: $base-color;
+    border-style: solid;
+    border-width: 4px;
+    border-radius: 4px;
+  }
```

Comment:
You can get the same style without below codes. 👍 
```scss
&::before,
&::after {
  ...
}

&::before {
  ...
}
&::after {
  ...
}
```

I think that it is better to substitute the following code instead of above codes. 🙏 
```scss
.nes-dialog {
  padding: 1.5rem 2rem;
  color: $base-color;
  border: none;
  box-shadow: 4px 0, -4px 0, 0 4px, 0 -4px;

  > :last-child {
    margin-bottom: 0;
  }
}
```
![2018-12-12 10 38 37](https://user-images.githubusercontent.com/5305599/49841216-1c1b4280-fdfa-11e8-875b-6a688b2be563.png)